arbitrum-docs/partials/_troubleshooting-bridging-partial.md

```diff
@@ -7,7 +7,8 @@
 ### What fees do I have to pay when bridging funds from L1 to L2?
 <p>When bridging over tokens from L1 to L2, you will have to sign one or two transactions with their corresponding fees:</p>
 
-<ol><li>If you are bridging a token for the first time, you'll sign one <strong>approval transaction</strong>.</li>
+<ol>
```

Comment:
These files are automatically generated from contents that live on Notion. When generating them again, and formatting, the same error appears here, so we would need to find a solution for this.